api/users/views.py

```diff
@@ -14,16 +16,35 @@
     OrganisationUsersPermission,
     UserPermissionGroupPermission,
 )
-from organisations.serializers import (
-    UserOrganisationSerializer,
-)
+from organisations.serializers import UserOrganisationSerializer
 from users.models import FFAdminUser, UserPermissionGroup
 from users.serializers import (
     UserIdsSerializer,
     UserListSerializer,
     UserPermissionGroupSerializerDetail,
 )
 
+from .forms import InitConfigForm
+
+
+class InitialConfigurationView(PermissionRequiredMixin, FormView):
+    template_name = "users/onboard.html"
+    form_class = InitConfigForm
+    permission_denied_message = (
+        "FAILED TO INIT Configuration. USER(S) ALREADY EXIST IN SYSTEM."
+    )
+
+    def has_permission(self):
```

Comment:
Nice! I like this. Keeping all this bespoke functionality nice and neatly in one place 👏